lnbits/wallets/macaroon/macaroon.py

```diff
@@ -2,6 +2,7 @@
 
 from loguru import logger
 
+from lnbits.settings import settings
```

Comment:
- settings should not be imported into this utility file
- better to read the secret from `input` (just as it is done with the `macaron` on line `42`)

Reply:
It's not viable to read the settings from input here as that causes a server error on startup.